src/megachatapi.h

```diff
@@ -2387,6 +2387,31 @@ class MegaChatApi
      */
     void setChatTitle(MegaChatHandle chatid, const char *title, MegaChatRequestListener *listener = NULL);
 
+    /**
+     * @brief Allows any user to preview an open chat without be part of.
+     *
+     * The associated request type with this request is MegaChatRequest::TYPE_PREVIEW_CHAT_LINK
+     * Valid data in the MegaChatRequest object received on callbacks:
```

Comment:
The following 2 lines are incorrect. It's only true when request success, not for all callbacks. I will amend it. The `getText()` should return the link in all callbacks. We should use a different field to return the title if success.